package.xml

```diff
@@ -8,7 +8,7 @@
   <description>
     Wrapper around mimick, it provides an ExternalProject build of mimick.
   </description>
-  <maintainer email="michel@ekumenlabs.com">Michel Hidalgo</maintainer>
```

Comment:
Same comment as https://github.com/ros2/tinyxml_vendor/pull/25#discussion_r744710444

Reply:
Done in e175b7e